src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURESCOUNT = 5;
+    private final Figure circleWhite10 = new Circle(Color.WHITE, 10);
```

Comment:
```
private static final int DEFAULT_RADIUS = 10;
private static final Color DEFAULT_COLOR = Color.WHITE;

public Circle getDefaultCircle() {
    return new Circle(DEFAULT_COLOR, DEFAULT_RADIUS);
}
```
Mentor suggested this kind of change, but your option is also viable I guess. `defaultCircle` would be better naming